pages/login.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Games Cart</title>
+    <meta name="author" content="Rohit Prasad" />
+    <meta name="description" content="Login" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="https://particle-ui.netlify.app/main.css" />
+    <link rel="stylesheet" href="../style/authentication.css" />
+    <link rel="stylesheet" href="../style/home.css" />
+  </head>
+
+  <body>
+    <div class="home__container">
+      <nav class="navbar simple">
+        <h3 class="navbar__header">Games-Cart</h3>
+        <ul class="navbar__list">
+          <li class="navbar__list__items">
+            <input type="text" class="search" placeholder="🔍 Search" />
+          </li>
+          <li class="navbar__list__items">
+            <button class="btn btn__secondary">
+              <a href="./login.html">Login</a>
+            </button>
```

Comment:
enclose button inside anchor tag ...otherwise when I am clicking on side of buttons the link isn't working ...only when we click in the middle of the button it works